ddtrace/tracer.py

```diff
@@ -79,7 +80,8 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None, contex
         :param object sampler: A custom Sampler instance
         :param object context_provider: The ``ContextProvider`` that will be used to retrieve
             automatically the current call context
-
+        :param object wrap_executor: callable that is used when a function is decorated with
```

Comment:
We could add a note that `context_provider`/`wrap_executor` are advanced options that the user should not modify.